CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 20.10.3
+
+### Additions and Improvements
+
+### Bug Fixes
+
+#### Previously identified known issues
+
+- [Eth/65 loses peers](KNOWN_ISSUES.md#eth65-loses-peers)
```

Comment:
@shemnon this one should not be here, as it has been removed from the known issues file due to Karim's fix.

Reply:
done